filters/ambiguouscharacters/filter.py

```diff
@@ -0,0 +1,26 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class AmbiguousCharactersFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+    keywords = ["highly-meaning-preserving", "low-generations", "rule-based"]
+
+    def __init__(self, keywords=None):
+        super().__init__()
+        self.ambiguous_chars = [
+                                '0', 'O', 'D', 'o', 'Q',
+                                'l', '1', 'I', 'i', '!', '|',
+                                'B', '8',
+                                'Z', '2',
+                                'S', '5',
+                                'G', '6',
+                                "'", '`',
+                                ]
+
+    def filter(self, sentence: str = None) -> bool:
+        for c in sentence:
+            if c in self.ambiguous_chars:
+                return False
```

Comment:
Hi @motiwari , I like the idea of ambiguouscharacters filter. But I have the same question as @ashish3586 . After I check out your PR and run it locally, I cannot pass all the test cases with the following error.
```
E           AssertionError: Executing AmbiguousCharactersFilter The filter should return True for the inputs {'sentence': 'She has a cat.'}
E           assert False == True
```
If I understand it correctly, 'She has a cat.' returned False since it contains ambiguous_chars 'S' .

Reply:
Thanks @Yiwen-Shi -- I have fixed that test case.

And @Yiwen-Shi and @ashish3586 your understandings are correct; for this filter, *any* sentence with a single ambiguous character will get filtered. This is a crude, first-pass filter that might be valuable to the community.

For future work, it would be interesting to filter sentences *only if* the ambiguous character actually leads to a difference in interpretation of the sentence, e.g. have some dictionary of English words and consider all possible substitutions of the ambiguous characters that could create multiple words. For example:

`piIIs` returns "ambiguous" because the capital `i`s can be changed to `l` to create a new (valid) English word
`She` would return "not ambiguous" because the `S` cannot be changed to a `5` to create a new English word

Unfortunately such an approach is significantly more complex and I won't be able to implement it in this PR; however, I hope that someone will build off of this idea to implement it